packages/rocketchat-slashcommands-unflip/unflip.js

```diff
@@ -0,0 +1,20 @@
+/*
+* Unflip is a named function that will replace /unflip commands
+* @param {Object} message - The message object
+*/
+
+
+function Unflip(command, params, item) {
+	if (command === 'Unflip') {
+		var msg;
+
+		msg = item;
+		msg.msg = params + ' ¯(╯°□°）╯︵ ┻━┻';
+		Meteor.call('sendMessage', msg);
+	}
+}
+
+RocketChat.slashCommands.add('Unflip', Shrug, {
+	description: 'Slash_Tableflip_Description',
```

Comment:
You'll need to actually add the i18n files that include the description like done with shrug, as well as change this to a different description.

This text is what shows up in the autocomplete box to tell the user what doing this command will output.